packages/styles/components/_c.checkbox.scss

```diff
@@ -12,6 +12,11 @@
     margin-left: magic-unit-rem(0.5, true);
   }
 
+  &__suffix-label {
+    color: $color-grey-600;
```

Comment:
It is preferable to use the variable that refers to the accessible text colour.:
~~$color-grey-600~~ => `$color-font-light`